tests/generative/variable_strategies.py

```diff
@@ -183,12 +200,84 @@ def subtract(draw, type_, frame):
     def multiply(draw, type_, frame):
         return draw(binary_operation(type_, frame, Function.Multiply))
 
+    @st.composite
+    def date_add_years(draw, type_, frame):
+        return draw(
+            binary_operation_with_types(type_, int, frame, Function.DateAddYears)
+        )
```

Comment:
No need for `@st.composite` here or in any of the functions below that do `return draw(...)`. If there is only one `draw()`, it can usually be eliminated. In this case you can write it as:

```
    def date_add_years(draw, type_, frame):
        return binary_operation_with_types(type_, int, frame, Function.DateAddYears)
```